### PR TITLE
feat: add support for 0.01% tier

### PR DESCRIPTION
**Pull request by justindomingue**
* in routing diagram for routes returned by the routing api
* in add liquidity

(assuming governance proposal passes)

todo: make router change per chain

### CODE REVIEW

File: src/components/FeeSelector/shared.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Trans } from '@lingui/macro'
+import { FeeAmount } from '@uniswap/v3-sdk'
+import { SupportedChainId } from 'constants/chains'
+import { ReactNode } from 'react'
+
+export const FEE_AMOUNT_DETAIL: Record<
+  FeeAmount,
+  { label: string; description: ReactNode; supportedChains: SupportedChainId[] | 'all' }
+> = {
+  [FeeAmount.LOWEST]: {
```
Comment (ianlapham) [inline src/components/FeeSelector/shared.tsx:10]: Would it be better to label them after bits?

**Review by ianlapham — COMMENTED** [review]
just small nit - rest looks good

File: src/components/FeeSelector/shared.tsx (hunk 1 above)
Comment (justindomingue) [reply src/components/FeeSelector/shared.tsx:10]: probably 😓 that would have to be done in the v3-sdk though :/

Comment (ianlapham) [reply src/components/FeeSelector/shared.tsx:10]: Ah in that case probably not worth th refactor

File: src/components/FeeSelector/shared.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    label: '0.01',
+    description: <Trans>Best for very stable pairs.</Trans>,
+    supportedChains: [SupportedChainId.MAINNET],
+  },
+  [FeeAmount.LOW]: {
+    label: '0.05',
+    description: <Trans>Best for stable pairs.</Trans>,
+    supportedChains: 'all',
```
Comment (moodysalem) [inline src/components/FeeSelector/shared.tsx:18]: ALL_SUPPORTED_CHAIN_IDS?

File: yarn.lock [hunk 3]
```diff
@@ -4654,18 +4666,28 @@
...
     tiny-warning "^1.0.3"
 
+"@uniswap/v3-staker@1.0.0":
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/@uniswap/v3-staker/-/v3-staker-1.0.0.tgz#9a6915ec980852479dfc903f50baf822ff8fa66e"
+  integrity sha512-JV0Qc46Px5alvg6YWd+UIaGH9lDuYG/Js7ngxPit1SPaIP30AlVer1UYB7BRYeUVVxE+byUyIeN5jeQ7LLDjIw==
+  dependencies:
+    "@openzeppelin/contracts" "3.4.1-solc-0.7-2"
+    "@uniswap/v3-core" "1.0.0"
+    "@uniswap/v3-periphery" "^1.0.1"
```
Comment (moodysalem) [inline yarn.lock:4689]: lol periphery getting duplicated by this

File: src/components/FeeSelector/shared.tsx (hunk 2 above)
Comment (moodysalem) [reply src/components/FeeSelector/shared.tsx:18]: or just `allowedChains: undefined | SupportedChainId[]`

Comment (moodysalem) [reply src/components/FeeSelector/shared.tsx:18]: one thing to consider is tick spacing may differ between the fee amounts on different chains. not sure where tick spacing is coming from here, is it being queried from the factory?

Comment (justindomingue) [reply src/components/FeeSelector/shared.tsx:18]: TICK_SPACINGS is updated here: https://github.com/Uniswap/v3-sdk/blob/main/src/constants.ts#L25

Comment (moodysalem) [reply src/components/FeeSelector/shared.tsx:18]: that constant already exists in the chains constants file

i'm also fine with `'all'`

Comment (justindomingue) [reply src/components/FeeSelector/shared.tsx:18]: oh nice, i prefer `ALL_SUPPORTED_CHAIN_IDS` if it's already defined.

**Review by Magkoooh — APPROVED** [review]

**Review by Magkoooh — APPROVED** [review]

**Pull request merged**